consul/base.py

```diff
@@ -1933,6 +1943,119 @@ class ACL(object):
         def __init__(self, agent):
             self.agent = agent
 
+        def create_token(self, accessor_id=None, secret_id=None, description=None, policies=None, roles=None, service_identities=None, local=None, expiration_time=None, expiration_ttl=None):
+            return self.agent.http.put(
+                CB.json(), '/v1/acl/token', params=args_to_payload(locals()))
+
+        def read_token(self, accessor_id):
+            return self.agent.http.get(
+                CB.json(), '/v1/acl/token/{}'.format(accessor_id))
+
+        def read_self_token(self):
+            return self.agent.http.get(
+                CB.json(), '/v1/acl/token/self')
+
+        def update_token(self, accessor_id, secret_id=None, description=None, policies=None, roles=None, service_identities=None, local=None, expiration_time=None, expiration_ttl=None):
+            return self.agent.http.put(
+                CB.json(), '/v1/acl/policies/{}'.format(accessor_id), params=args_to_payload(locals()))
```

Comment:
Path should be `/v1/acl/token/{}`

Reply:
ok @abarbare I'll fix that today or tomorrow